setup.py

```diff
@@ -15,7 +15,15 @@
     author='Rocio Aramberri Schegel',
     author_email='rocioaramberri@schegel.net',
     url='http://github.com/rocioar/flake8-django',
-    py_modules=['flake8_django', 'issues', 'checkers'],
+    py_modules=[
+        'flake8_django',
+        'checkers',
+        'checkers.checker',
```

Comment:
It looks like checkers is a package, so should be listed as `packages=['checkers']` rather than modules.